debian/12/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM debian:12
```

Comment:
Would it make sense to use the `12-slim` tag here, or is that too much of a change?